src/app/components/pages/wallets/wallets.component.html

```diff
@@ -10,7 +10,7 @@
     <div class="-wallets">
       <ng-container *ngFor="let wallet of wallets">
         <div class="-wallet" (click)="toggleWallet(wallet)">
-          <div class="-width-250 -label">{{ wallet.label }}</div>
+          <div class="-width-250 -label" [attr.title]="wallet.label.length > 36 ? wallet.label : ''">{{ wallet.label }}</div>
```

Comment:
Please remove the 36 characters condition. If the name has many wide letters (like "M"), 27 characters could be enough to show the ellipsis, but if it has many narrow letters (like "i"), 47 characters fit without problems. As it is so difficult to know how many characters fit in the space, it seems that it is best to show the tooltip always, instead of risking to have an inconsistent UI.

Reply:
Fixed.